3868 docs spark uplevel (#3871)
treeverse/lakeFS: docs/spark/index.md

---
layout: default
title: Spark

> description: Spark top level. This section priovids indexing for finding how to integrate lakeFS with Spark. As well as using dedicated table formats such as hudi, iceberg and deltalake.

Reviewer: ```suggestion
description: Spark top level. This section indexes finding how to integrate lakeFS with Spark, and also how to use dedicated table formats such as Hudi, Iceberg and Delta Lake.
```

nav_order: 8
has_children: false
---

# lakeFS with Spark:
{: .no_toc }

## Looking to run lakeFS with Apache Spark 💫 ?  

Visit [Spark-lakeFS integration page](../integrations/spark.md).

> Want to integrate lakeFS with Spark and DeltaLake? Checkout [Configuring DeltaLake with lakeFS](../integrations/delta.md).

Reviewer 2: This is pretty specific... I think that the mapping should be the other way around, i.e. figure out which integration pages are most likely related to Spark and link (VISIBLY) to the lakeFS Hadoop Filesystem integration page.

Author: Thanks, Jonathan, do you mean to link to this - [#two-tiered-spark-support](https://docs.lakefs.io/integrations/spark.html#two-tiered-spark-support) instead of the whole spark page?

Author: updated! Please check now.

> Want to integrate lakeFS with Spark and DeltaLake? Checkout [Configuring DeltaLake with lakeFS](../integrations/delta.md).

Reviewer: ```suggestion
## Want to integrate lakeFS with Spark and DeltaLake?

Check out [Configuring DeltaLake with lakeFS](../integrations/delta.md).
```

Author: updated!

Have questions? want to contribute new features? 
💻 Join the [conversation!](https://lakefs.slack.com/).

>   

Reviewer: Probably want to remove these spaces.
